README.md

```diff
@@ -0,0 +1,97 @@
+heroImagePath: ../assets/scheduler-hero-2.png
+authors:
+  - Vikram Koka
+date: 2020-12-17T16:17:00.000Z
+---
+```
+
+Fill in all the fields shown above with information relative to your post. The `title` field will make a h1 tag at the top of the post with your title. Do not use h1s in your `.md` file. The `slug` field is used as the as the path to the post. The `heroImagePath` field is the path to the hero image. Hero images are stored in the `assets` directory. If you do not have a hero image use `../assets/blank.jpg`. 
+
+Images used in the blog posts are stored in the `assets` directory in this repo. Below is an example of how to include an image in your markdown folder. 
+
+```markdown
+![Airflow Scheduler Deployment Model](../assets/airflow-2-scheduler/scheduler-deployment-model.png)
+```
+
+Please follow standard markdown formatting rules that are outlined [here](https://github.com/DavidAnson/markdownlint/blob/main/doc/Rules.md) more information can be found below in the markdown Lint section.
+
+### Netlify CMS
+
+You can also use Netlify's content management system to create blog posts. The CMS tool for our blog is located at [https://astro-blog.netlify.app/admin](https://astro-blog.netlify.app/admin). When creating a post you will be prompted to provide the `frontmatter` information described above. The CMS tool will create a PR to this repo.
+
+To log into the CMS go to `/admin` and login with your github account. You will need to have access to the `astronomer/astro-blog` repo to login to the CMS. 
+
+## CI/CD Pipeline
+
+A GitHub Actions CI/CD Pipeline is used to verify each markdown file is free from formatting errors, spelling errors, and broken links. To see logs from this pipeline go to `Actions` section of the GitHub repo. Run the commands outlined below locally to fix errors before contributing posts.
+
+Fix pipeline errors from all posts as old links may break and need to be replaced.
+
+### Markdown Lint
+
+The pipeline uses [markdownlint](https://github.com/DavidAnson/markdownlint) to ensure proper markdown formatting to ensure consistency. The rules followed by the linter can be found [here](https://github.com/DavidAnson/markdownlint/blob/main/doc/Rules.md). To run the linter locally run the following.
+
+```shell
+node_modules/.bin/markdownlint ./posts/<your-blog-post> --ignore node_modules --ignore v5
+```
+
+To include lines of code that break certain markdown rules follow the code example below.
+
+```markdown
+<!-- markdownlint-disable MD033 -->
+<hr/>
+```
+
+### Spell Check
+
+The pipeline features a [spell checker](https://github.com/lukeapage/node-markdown-spellcheck) to ensure that there are no misspelled words in our blog posts. You may want to run the spell checker locally in "interactive" mode to add proper nouns to the dictionary.
+
+```shell
+node_modules/.bin/mdspell "posts/<your-blog-post>.md" -n -a --en-us
```

Comment:
Seems like we could do something similar here by adding a Yarn script?
```suggestion
yarn spellcheck
```

Reply:
running this command locally is different than in the pipeline. In the pipeline it would be run  in report mode locally it would be run in interactive mode. Maybe We can make another yarn script called `yarn spellcheck-interactive`